.changeset/giant-tools-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spotlightjs/overlay": minor
```

Comment:
Unless this adds a new feature or improves the end-user experience in some way, this should be a patch.

```suggestion
"@spotlightjs/overlay": patch
```